projects/storefrontlib/cms-components/myaccount/my-coupons/claim-dialog/claim-dialog.component.html

```diff
@@ -0,0 +1,102 @@
+<div
+  class="cx-coupon-dialog"
+  role="dialog"
+  [cxFocus]="focusConfig"
+  (esc)="close('Escape clicked')"
+  aria-labelledby="dialogTitle"
+>
+  <div class="cx-coupon-container">
+    <!-- Modal Header -->
+
+    <div class="cx-dialog-header modal-header">
+      <h3
+        *cxFeature="'a11yDialogsHeading'"
```

Comment:
MAJOR: It seems like this code was copied from somewhere and includes redundant feature toggles.
It is a new component, so you don't need any template parts hidden behind *cxFeature directives, as well as templates for cases when feature toggle is not enabled as *cxFeature="'!a11yDialogsHeading'". 
When we change html of already released components, it could break customers logic, so we are using such mechanisms
```html
<h3
        *cxFeature="'a11yDialogsHeading'"
        id="dialogTitle"
        class="cx-dialog-title modal-title"
      >
        {{ 'configurator.overviewFilter.title' | cxTranslate }}
      </h3>
      <div
        *cxFeature="'!a11yDialogsHeading'"
        id="dialogTitle"
        class="cx-dialog-title modal-title"
      >
        {{ 'configurator.overviewFilter.title' | cxTranslate }}
      </div>
``` 
But in your case you could just use h3 without  *cxFeature="'a11yDialogsHeading'"

```html
<h3
        id="dialogTitle"
        class="cx-dialog-title modal-title"
      >
        {{ 'configurator.overviewFilter.title' | cxTranslate }}
</h3>
```

Reply:
thanks, fixed.